### ISSUE TITLE
Evaluation set

### ISSUE THREAD

**Issue by yangwang4040** [rhgao/co-separation#5]
To Calculate SDR, SIR and SAR, did you consider two videos only? Or consider whole test set. Because evaluateSeparation.py considers two audios only. It's not considering whole test set.

**Comment by rhgao** [comment]
evaluateSeparation.py considers two videos only. You can evaluate across the test set and get the mean SDR/SIR/SAR.

**Comment by avis-ma** [comment]
Hi, I want to know about the test strategy, in the process of testing, two samples are selected from the test set. How to select the two samples? To ensure that the samples are from different categories or two samples under the same category can also be selected? What selection strategies are used to calculate the results in your paper?  @rhgao

**Comment by rhgao** [comment]
@avis-ma Hi, for MUSIC, the first/second video in each category are used as validation/test data, and the rest as training data. The testing is done by randomly mixing two single-source videos for the 11 test videos (each for one class, 550 test pairing). For AudioSet, we randomly mix and separate two videos from AudioSet-SingleSource dataset here: http://vision.cs.utexas.edu/projects/separating_object_sounds/

**Comment by avis-ma** [comment]
Hi, you mean that , there are 11 categories, each category has a video, and a video is divided into multiple segments (10s one segment). During the test, one segment is randomly selected from one category, and then one segment is selected from the other 10 categories to form 10 groups of test pairs. Then a total of 55 groups of such test pairs were selected. Is there any deviation in my understanding？

**Comment by rhgao** [comment]
That is correct. I forgot to mention 10 groups. We randomly sample one 10s segment for each category for each group. For each group of 11 testing segments, there are 55 pairs. 10 groups x 55 pairs = 550 pairs in total.

**Comment by avis-ma** [comment]
OK, thank you very much. : )
